CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Release 0.11.0
```

Comment:
Changelog looks good, thank you.  Pretty small release!

Reply:
Still too big, I think.